build.gradle

```diff
@@ -132,5 +132,5 @@ apply from: "licenses-source-header.gradle"
 ext {
     publicDir =  "${project.rootDir}"
     neo4jVersionEffective = project.hasProperty("neo4jVersionOverride") ? project.getProperty("neo4jVersionOverride") : "5.9.0"
-    testContainersVersion = '1.17.6'
+    testContainersVersion = '1.18.3'
```

Comment:
If I search for 1.17.6 I get among other hits, this line in LICENSES.txt:

testcontainers-1.17.6.jar

Do we need to do anything so the license files are updated when we update dependencies and the build fails otherwise, like in the monorepo? cc @gem-neo4j

Reply:
Yes, good point. The build is meant to fail on new files being added without a license header as well (although this seems to only fail locally with ./gradlew build as I just tried it and it failed but this PR is fine 🤔). I'll look into this when I have time!

Reply:
Ok maybe we can create a card for both those things?

Reply:
https://trello.com/c/VpC8nqNz/2264-add-build-failure-on-license-file-change

Reply:
I know what's failing, I can fix it in a separate pr. Thanks for spotting it 🤗

Reply:
I think @gem-neo4j is looking into it already